src/detect_breakends/junction_detection.cpp

```diff
@@ -283,67 +283,64 @@ void detect_junctions_in_alignment_file(const std::filesystem::path & alignment_
     {
         const std::string query_name            = seqan3::get<seqan3::field::id>(rec);                      // 1: QNAME
         const seqan3::sam_flag flag             = seqan3::get<seqan3::field::flag>(rec);                    // 2: FLAG
-        const int32_t ref_id                    = seqan3::get<seqan3::field::ref_id>(rec).value_or(0);      // 3: RNAME
+        const int32_t ref_id                    = seqan3::get<seqan3::field::ref_id>(rec).value_or(-1);     // 3: RNAME
         const int32_t pos                       = seqan3::get<seqan3::field::ref_offset>(rec).value_or(0);  // 4: POS
```

Comment:
According to the [seqan docs](https://docs.seqan.de/seqan/3-master-dev/group__io.html#ga2511b22375848e1bedbc5013c952a817) both ref_id and pos are 0-based. Therefore, you can also set a missing pos to -1:

```suggestion
        const int32_t pos                       = seqan3::get<seqan3::field::ref_offset>(rec).value_or(-1);  // 4: POS
```